Dockerfile

```diff
@@ -105,12 +105,13 @@ RUN npm install
 COPY ./ ./
 
 # Jekyl ports
-EXPOSE 35729
+EXPOSE 35730
 EXPOSE 4002
 
 # Firebase emulator port
 # Airplay runs on :5000 by default now
-EXPOSE 5500
+EXPOSE 5502
```

Comment:
We'll want to update this port value in the [firebase.json](https://github.com/flutter/website/blob/main/firebase.json#L447)

Reply:
Oddly enough, I don't. When I do that, the staging build fails.

Reply:
I just look a deeper dive and I think this port 5502 doesn't actually need to change to run Jekyll concurrently. 5500 is used for the Firebase emulator which is only used by the link checker (testing) Do you visit port 5502 for any reason? Otherwise, we might just be able to leave this port alone.